src/Utility.cc

```diff
@@ -990,10 +967,17 @@ namespace ignition
       // Get the top level <model> element.
       tinyxml2::XMLElement *modelElement = modelConfigDoc.FirstChildElement(
           "model");
+      bool isModel = true;
       if (!modelElement)
       {
-        std::cerr << "Model config string does not contain a <model> element\n";
-        return false;
+        modelElement = modelConfigDoc.FirstChildElement("world");
```

Comment:
Nitpick that this isn't a "model" element anymore. The variable could be updated to be more generic, and the comment in line 967 could mention `<world>` as well.

Reply:
Done c86f6d101a65e5dd7ac18664fe5da3f78bd71a1a :smile: